rosidl_generator_c/src/message_bounds.c

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rosidl_generator_c/message_bounds_struct.h"
+
+#include <assert.h>
+#include <stdio.h>
+#include <string.h>
+
+const rosidl_message_bounds_t * get_message_bounds_handle(
+  const rosidl_message_bounds_t * handle, const char * identifier)
+{
+  assert(handle);
```

Comment:
I know this is following a pattern from the rest of the code, but it seems like this is encoding an error policy down in this library.  I won't insist that you fix it here, but I would appreciate opening an issue to revisit the use of `assert` in this entire library.

Reply:
This is a direct copy of what is currently happening in `message_type_support.c`.  I can open an issue to address both cases.